resources/.tsan-ignore.txt

```diff
@@ -12,12 +12,18 @@ race:^std::__1::unique_ptr
 race:^std::__1::__unique_if
 race:shared_count
 
+race:^std::atomic
+
 # Boost is not built with tsan support, either
 race:boost::detail::thread_data_base
 race:thread_data
 
 # A DebugAssert makes sure that predecessors are not added to scheduled tasks, but it is missing in release builds
 race:^opossum::AbstractTask::set_as_predecessor_of
 
+# tsan does not understand parts of our MVCC concept - see mvcc_data.hpp for documentation
+race:&opossum::MvccData::set_begin_cid
+race:&opossum::MvccData::set_end_cid
```

Comment:
I have not used tsan before, but '&' does not seem to be a valid token: https://github.com/google/sanitizers/wiki/ThreadSanitizerSuppressions
If I locally replace it with '^', the listed methods get actually filtered.

Reply:
But the keys are right next to each other. That should count for something, right?